var/spack/repos/builtin/packages/parsec/package.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+#
+# Copyright (c) 2017-2020  The University of Tennessee and the University
+#                          of Tennessee Research Foundation.  All rights
+#                          reserved.
+#
```

Comment:
Why the additional copyright? I think this is unique in the built-in repository. Can we just use the standard Spack header?